src/test/java/racingcar/Car.kt

```diff
@@ -0,0 +1,19 @@
+package racingcar
+
+class Car(
+    val name: String,
+) {
+    private var _position: Int = 0
+    val position: Int
+        get() = _position
```

Comment:
```suggestion
    val position: Int
        private set
```

어떤 부분이 다를까요? Backing Properties 형태의 문법을 사용할 필요가 있을지 고민해보시면 좋겠습니다.
굳이 프로퍼티를 두개로 늘려서 복잡도를 높일 필요는 없어보여요. private set만 사용해도 외부에서 수정할 수 없고 대신 조회는 가능한 요구사항을 충족할 수 있습니다.

혹시 다른 의도가 있다면 코멘트 남겨주세요~